doc/rst/developer/bestPractices/TestSystem.rst

```diff
@@ -232,24 +437,9 @@ verify that the last line of output contains the string "SUCCESS":
     Memory:
     verify:-1: SUCCESS
 
-
-Invoking start_test for performance testing
--------------------------------------------
-To run performance testing, add the ``--performance`` flag to ``start_test``
-along with the traditional options.  So for example, to run this
-single test in performance mode, one could use:
-
-  ``start_test --performance foo.chpl``
-
-When crawling a directory hierarchy, only tests with ``.perfkeys`` files
-will be considered when testing in performance mode.
-
-All performance tests are compiled with ``--fast`` by default and ``--static``
-when it's not problematic for the target configuration.
-
-
 Accumulating Performance Data in .dat files
```

Comment:
I think the description of the ``.dat`` file could be improved by being more precise.  It starts off with "the values collected ... are stored as a tab-delimited ``.dat`` file", coming back a few sentences later to say that each run is its own line.   I think it would be better if it said that in the first place.  I get the impression from the first sentence that the file is overwritten by each run, even though I know that's not how it works.  Maybe move that last sentence of the paragraph to the beginning, and merge it with the first one?

Also, it'd help to understand the graphing to say that the first entry on each line is the date the test was run.

Reply:
Mentioning that the date's on each line will help make sense of the example ``graph`` file later,

> ylabel: Time (seconds)

> ylabel: a label for the graph's y-axis (the x-axis will be time by default)

If you can refer to the date the test was run, it'd be clearer that it's not the same "time" on both axes.